piccolo_api/encryption/providers.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import annotations
+
+import logging
+import typing as t
+from abc import ABCMeta, abstractmethod
+
+import cryptography.fernet
+import nacl.encoding
+import nacl.secret
```

Comment:
Importing these here negates `get_cryptography` raising a helpful error as line 7 will throw the relevant missing error before a user has the opportunity to hit it

Reply:
@dantownsend Just one thing. I just tried the Piccolo Admin MFA branch with newest Piccolo API version and removing those imports breaks the MFA flow. The error log is
```bash
File "/home/rkl/dev/admin_env/lib/python3.11/site-packages/piccolo_api/mfa/endpoints.py", line 126, in post
    html_content = await self._provider.get_registration_html(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/rkl/dev/admin_env/lib/python3.11/site-packages/piccolo_api/mfa/authenticator/provider.py", line 121, in get_registration_html
  secret, recovery_codes = await self.secret_table.create_new(
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/rkl/dev/admin_env/lib/python3.11/site-packages/piccolo_api/mfa/authenticator/tables.py", line 116, in create_new
  encrypted_secret = encryption_provider.encrypt(value=secret)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/rkl/dev/admin_env/lib/python3.11/site-packages/piccolo_api/encryption/providers.py", line 122, in encrypt
  fernet = cryptography.fernet.Fernet(  # type: ignore
            ^^^^^^^^^^^^^^^^^^^
AttributeError: module 'cryptography' has no attribute 'fernet'
```
If I patch the local Piccolo API installation with those imports, everything works.

Reply:
Ah, good catch - I'll do a new release later today.